bin/process_snaps.py

```diff
@@ -89,7 +98,8 @@
         "beta": {"recipe": "mesa-core22-beta"},
     },
     "mesa-2404": {
-        "beta": {"recipe": "mesa-2404-beta"},
+        "stable": {"recipe": "mesa-2404-beta", "check-usns": False, **MESA_2404_PATHS},
+        "beta": {"recipe": "mesa-2404-beta", **MESA_2404_PATHS},
```

Comment:
It might be a little annoying if they differ in beta... OTOH that would give us notice that they need updating.